libraries/chain/protocol/transaction.cpp

```diff
@@ -303,6 +303,15 @@ void verify_authority( const vector<operation>& ops, const flat_set<public_key_t
 
 
 flat_set<public_key_type> signed_transaction::get_signature_keys( const chain_id_type& chain_id )const
```

Comment:
Can change method signature to return a const reference to the set, to avoid useless copying.